src/core_modules/capture-core/components/DataEntries/common/TEIAndEnrollment/useMetadataForRegistrationForm/hooks/useDataEntryFormConfig.ts

```diff
@@ -9,17 +9,19 @@ const configQuery = {
 };
 
 export const useDataEntryFormConfig = ({ selectedScopeId }: Props) => {
-    const { data: dataEntryFormConfig, isFetched: configIsFetched } = useApiMetadataQuery(
+    const { data: dataEntryFormConfig, isFetched: configIsFetched, isInitialLoading } = useApiMetadataQuery(
         ['dataEntryFormConfig', selectedScopeId],
         configQuery,
         {
             enabled: !!selectedScopeId,
             select: (dataEntryFormConfigQuery: any) => dataEntryFormConfigQuery[selectedScopeId],
+            retry: false,
```

Comment:
Should this be added as a default on the useApiMetadataQuery perhaps? Does it make sense to retry if there is an error?